internal/cli/atlas/deployments/list.go

```diff
@@ -71,6 +71,7 @@ func (opts *ListOpts) Run(ctx context.Context) error {
 }
 
 func (opts *ListOpts) PostRun() error {
+	opts.UpdateDeploymentTelemetry()
```

Comment:
what is the expectation for list here?

Reply:
it will only set it if customer provides --type local/atlas, otherwise it will not set the type